app/pages/5_Model_assumptions.py

```diff
@@ -0,0 +1,83 @@
+
+import streamlit as st
+import pandas as pd
+import geopandas as gpd
+import plotly.subplots as sp
+import plotly.graph_objects as go
+
+st.title("Modelling assumptions")
+
+st.header("Assumptions for technology costs")
+
+st.markdown(
+    "Model assumptions are under constant revision."
+    "Assumptions used for this study are basted on the "
+    "[technology-data repository v0.6.2](https://github.com/PyPSA/technology-data/blob/master/outputs/costs_2030.csv). "
+    "The most relevant technologies are outlined below:"
+    ""
+)
+
+url = "https://raw.githubusercontent.com/PyPSA/technology-data/master/outputs/costs_2030.csv"
+costs = pd.read_csv(url, index_col=0)
+
+considered = [
+    'central air-sourced heat pump', 'decentral air-sourced heat pump',
+    'central ground-sourced heat pump', 'decentral ground-sourced heat pump',
+    'central resistive heater', 'decentral resistive heater',
+    'central solar thermal', 'decentral solar thermal',
+    'central solid biomass CHP',
+    'central water tank storage', 'decentral water tank storage',
+    'water tank charger', 'water tank discharger',
+
+    'battery inverter', 'battery storage',
+    'biogas upgrading', 'direct air capture', 'electricity distribution grid', 'electricity grid connection',
+    'electrolysis', 'fuel cell', 'central gas CHP', 'central gas boiler',
+    'decentral gas boiler', 'hydrogen storage underground', 'hydrogen storage tank type 1',
+    'HVDC inverter pair', 'helmeth', 'micro CHP', 'OCGT', 'offwind',
+    'onwind', 'PHS', 'ror', 'methanation', 'solar-rooftop',
+    'solar-utility', 'Steam methane reforming'
+]
+
+st.write(costs.loc[considered])
+
+st.write("FOM: Fixed operation and maintenance costs. Given as percentage of overnight costs per annum.")
+st.write("VOM: Variable operation and maintenance")
```

Comment:
I'd probably add * and ** to give a feeling that those are kind of footnotes. Or would give clear it in any other way that here we are actually explaining the abbreviations. What do you think?